client/src/pages/ProfilePage/MiniPortfolio/Project/projectItem.js

```diff
@@ -0,0 +1,126 @@
+import * as React from "react";
+import { Card, CardContent, Typography, Grid } from "@mui/material";
+let displayEnddate = (end_month, end_year) => {
+  return (
+    <>
+      {displayMonthInWord(end_month)}
+      {end_year}
+    </>
+  );
+};
+let displayMonthInWord = (month) => {
```

Comment:
I see this being used in two places. Move this to a utils file and have the component import to use it